specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/managedDatabases.json

```diff
@@ -0,0 +1,556 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-03-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/managedDatabaseRestoreAzureAsyncOperation/{operationId}/completeRestore": {
+      "post": {
+        "tags": [
+          "ManagedDatabases"
+        ],
+        "description": "Completes the restore operation on a managed database.",
+        "operationId": "ManagedDatabases_CompleteRestore",
+        "parameters": [
+          {
+            "name": "locationName",
+            "in": "path",
+            "description": "The name of the region where the resource is located.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "description": "Management operation id that this request tries to complete.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The definition for completing the restore of this managed database.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedDatabase"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully issued complete restore request."
+          },
+          "default": {
```

Comment:
>"default": { [](start = 10, length = 12)

Do you have a schema for default?

Reply:
What do you mean? I think this gets automatically generated based on AggregateErrorHandlers.

Reply:
Just a question in case you had something different

Reply:
@MarkoBender Add a schema for the default case to handle the error cases. The schema is defined as per the RPC. YOu can find an example here - https://github.com/Azure/azure-rest-api-specs/blob/current/specification/batch/resource-manager/Microsoft.Batch/2017-05-01/BatchManagement.json#L149

Reply:
I've fixed this based on example that you gave. Please take a look.